src/pages/Home.jsx

```diff
@@ -0,0 +1,10 @@
+
+const Home = () => {
+    return (
+        <>
```

Comment:
Heb je `<></>` hier wel nodig? De `<h1>`-tag is volgens mij voldoende als omwikkelend element?